spec/integration/rails/generators/graphql/install_generator_spec.rb

```diff
@@ -45,10 +45,22 @@ class DummySchema < GraphQL::Schema
   mutation(Types::MutationType)
   query(Types::QueryType)
 
+  # For batch-loading (see https://graphql-ruby.org/dataloader/overview.html)
+  use GraphQL::Dataloader
+
+  # GraphQL-Ruby calls this when something goes wrong while running a query:
+  def self.type_error(err)
```

Comment:
was the intent to remote the `ctx` parameter from `type_error`? the call to `super` results in an error due to the missing parameter for me (running v1.12.20)

Reply:
Just an oversight on my part 😖 getting a fix here: https://github.com/rmosolgo/graphql-ruby/pull/3722